### ISSUE TITLE
The underlying connection was closed: Could not establish trust relationship 

### ISSUE THREAD

**Issue by d-j-a** [pbatard/rufus#2223]
---------
- [x] I looked at https://github.com/pbatard/rufus/wiki/FAQ to see if my question has already been answered.
- [x] I performed a search in the issue tracker for similar issues using keywords relevant to my problem, such as the error message I got from the log.
- [x] I clicked the 'Log' button or pressed <kbd>Ctrl</kbd>-<kbd>L</kbd> in Rufus, or used [DebugView](https://learn.microsoft.com/en-us/sysinternals/downloads/debugview), and copy/pasted the log into the section that says `<FULL LOG>` below.
- [x] The log I am copying is the FULL log, starting with the line `Rufus version: x.y.z` - I have NOT removed any part of it.

Additionally (if applicable):
- [ ] I ran a bad blocks check, by clicking _Show advanced format options_ then _Check device for bad blocks_, and confirmed that my USB is not defective.
- [ ] I also tried one or more of the following:
  - [ ] Using a different USB drive.
  - [ ] Plugging the USB into a different port.
  - [ x] Running Rufus on a different computer. (6)
- [ ] If using an image, I clicked on the `(✓)` button to compute the MD5, SHA1 and SHA256 checksums, which are therefore present in the log I copied. I confirmed, by performing an internet search, that these values match the ones from the official image.

Issue description
-----------------
When attempting to download a Windows image on a device (PC) that WAS working (last week) I receive the error message:
_Error: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel._

I have tested on 5 other devices. No network changes have occurred since Rufus last worked for this task.  Video link: https://youtu.be/0rxnZfXt3_o
I have tried with no Windows Firewall running and AV disabled.  

I would like to note that on a device I am using Bitdefender on, it now flags the download of the script Fido.ps1.lzma with this warning:
_DotNet tried to load a malicious resource detected as Gen:Variant.MSILHeracles.75607 and was blocked. Your device is safe._

Log
---
```
Rufus x86 v3.22.2009
Windows version: Windows 11 Pro, 64-bit (Build 22621.730)
Syslinux versions: 4.07/2013-07-25, 6.04/pre1
Grub versions: 0.4.6a, 2.06
System locale ID: 0x0409 (en-US)
Will use default UI locale 0x0409
SetLGP: Successfully set NoDriveTypeAutorun policy to 0x0000009E
Localization set to 'en-US'
Found VHD device 'Microsoft Virtual Disk'
Removing C: from the list: This is the system disk!
0 devices found
Downloading https://github.com/pbatard/Fido/releases/download/v1.44/Fido.ps1.lzma
File length: 18.2 KB
Successfully downloaded 'Fido.ps1.lzma'
Download signature is valid ✓
Script signature is valid ✓
Please Wait...
Querying https://www.microsoft.com/en-US/software-download/
Querying https://vlscppe.microsoft.com/tags?org_id=y6jn8c31&session_id=8c715fb5-c64f-40fa-9fb0-cd4830540e70
Error: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel.
```

**Comment by d-j-a** [comment]
Additional note: the issue seems limited to Windows 10/11 ISOs

**Comment by pbatard** [comment]
> DotNet tried to load a malicious resource detected as Gen:Variant.MSILHeracles.75607 and was blocked. Your device is safe.

Please report this FALSE POSITIVE to bitdefender, pointing them to https://github.com/pbatard/Fido/blob/master/Fido.ps1 which is the exact script that's being compressed through LZMA and asking them __where exactly in this script they are seeing malicious behaviour__

> `Error: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel.`

I can't replicate your issue, so this has to be environmental. Especially, you seem to be running Windows off a VHD, which leads me to think that you're running on virtual hardware, and virtual networking tends to create all kind of issues.

Or you could have some firewall (that was added by your ISP) that prevents some connections. Or a PowerShell policy. Or a myriad of other environmental things that make it impossible to advise you what to do (short of trying from a completely different network, and from a clean install of Windows on real hardware).

Unless other people start to report a similar issue, I will have to dimiss it as a purely environmental matter that you will have to figure out on your own.

**Comment by d-j-a** [comment]
I reported the false positive yesterday.  However- that error you posted is NOT from a device with BitDefender. Just a standard Windows Defender cfg.  The AV was taking an issue with the download of Fido.ps1.lzma (See my note above in OP) 

I took the video and log from a HyperV device, but the issue is present on my laptop, which runs Windows 11 22H2 build 22624.1546. I'm running a PfSense FW and haven't changed anything- so I'm wondering what else there could be.  I assumed environmental issues at first as well.  I tested in other (non HV) VMs as well.  (UNRAID)
